### ISSUE TITLE
Suggestions for new SideQuest (new / updated apps)

### ISSUE THREAD

**Issue by crosstec** [the-expanse/SideQuest#491]
The new version is great but it doesn't reflect the growing popularity and engagement of the platform. 

For example: lesser popular apps currently seem to fly under the radar since the last update and could discourage further apps development which in turn affects user engagement.

Especially in growth-times, people need to realize how much new is actually going on. 
This keeps people sticking with the platform and helps to continue the growth (not just users but also devs).

Therefore I suggest the following simple steps:

- Homepage showing new apps first, with release dates (let's say up to 10 most recent new ones)

- Separate top-menu item for recently updated games (full list ordered by latest first), with update dates (shouldn't be buried in the filter options)

- Getting Started menu item, with video and text, explaining as simple as possible how to connect with Sidequest. Best displayed as corner-badge or something like that.

- Optional: automated chat-bot, e.g. to help setting up SideQuest.
Example: [https://space10-community.github.io/conversational-form/landingpage/](url)

The first ones should be relatively simple to achieve. The 2 latter ones requires some in-depth thinking on how to approach "normies" in a way to lower their entry-barriers.

**Comment by RedBrumbler** [comment]
the "chat bot" is fairly similiar to how the bot reacts to certain messages on the discord. Recently some errors also got a solution right when you get the error to make it more quick to resolve

**Comment by crosstec** [comment]
> the "chat bot" is fairly similiar to how the bot reacts to certain messages on the discord. Recently some errors also got a solution right when you get the error to make it more quick to resolve

Yeah, bot-stories basically. You could even add AI but I think that would be overkill.

**Comment by sidequestlegend** [comment]
Hi @crosstec !

Homepage showing new apps first, with release dates (let's say up to 10 most recent new ones)
**We have added the new section "Hot & Fresh" which has been working well to highlight new exciting content. We have also restored the Latest Releases section to the homepage so thats 12 apps on the homepage between those two sections.** 

Separate top-menu item for recently updated games (full list ordered by latest first), with update dates (shouldn't be buried in the filter options)
**This has been added and also broken out of the search modal, this should make it easier to filter the app results page.** 

Getting Started menu item, with video and text, explaining as simple as possible how to connect with Sidequest. Best displayed as corner-badge or something like that.
**The "Get SideQuest" button at the top is essentialy this feature. We have made the button more prominent recently too.** 

Optional: automated chat-bot, e.g. to help setting up SideQuest.
Example: https://space10-community.github.io/conversational-form/landingpage/
We have considered this, we are currently working on a much better wiki system to allow for easy searching and and will auto suggest solutions to common problems. We will look at ways to better integrate this into the website in the near future. - https://sidequestvr.atlassian.net/servicedesk/customer/portals
